lib/Code/TidyAll.pm

```diff
@@ -659,6 +669,10 @@ sub find_matched_files {
     my $root_length      = length( $self->root_dir );
     foreach my $plugin ( @{ $self->plugin_objects } ) {
         my @selected = grep { -f && !-l } $self->_zglob( $plugin->selects );
+        if ( @{ $self->global_ignore || [] } ) {
```

Comment:
If `global_ignore` had a default of `sub { [] }` you wouldn't need the `|| []` bit here.

Also, this could be combined with the check for the plugin ignores in one map statement and call to `$self->_zglob`.